lib/nostrum/struct/message/reference.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Nostrum.Struct.Message.Reference do
+  @moduledoc """
+  Struct representing a discord message reference.
+  """
+
+  alias Nostrum.{Snowflake, Util}
+  alias Nostrum.Struct.Message.{Channel, Guild, Message}
```

Comment:
I think this needs changed to something like this, right?
```suggestion
  alias Nostrum.Struct.{Channel, Guild, Message}
```

Reply:
Yep, my bad.